README.md

```diff
@@ -11,19 +11,31 @@
 
 ## fast.ai 코스 관련
 
-| 현황 | 원제목 | 번역제목 | 
-| ---- | ---- | ----   |
-| 진행중 | [Understanding Fastai's fit_one_cycle method]() | 미정 |
+| 현황 | 원글 | 번역글 | 
+| ---- | ------- | ------- |
+| 진행중 | [Understanding Fastai's fit_one_cycle method]() | [미정]() |
```

Comment:
Is there a reason you make a fake link
It's misleading to users.

Reply:
Actually I did not recognize the fake link. It's my fault.